src/models/Settings.js

```diff
@@ -156,7 +156,10 @@ function (Okta, Q, Errors, BrowserFeatures, Util, Logger, OAuth2Util, config) {
       'consent.cancel': 'function',
 
       //IDP Discovery
-      'idpDiscovery.requestContext': 'string'
+      'idpDiscovery.requestContext': 'string',
+
+      //Colors
+      'colors.brand': 'string'
```

Comment:
add readme and the color format

Reply:
Will add the format 👍 
For the README, we want to update when the new version that includes the feature is published.
CC: @jmelberg-okta

Reply:
Yep. Because we do not have a set release cadence, we've decided to separate README changes from the feature work. As an example, you can see the `afterEvents` documentation picked up with the `2.15.0` CHANGELOG PR (#605).